massa-bootstrap/src/server.rs

```diff
@@ -373,6 +411,75 @@ impl BootstrapServer {
     }
 }
 
+/// TODO: Doc-comment me
+#[allow(clippy::too_many_arguments)]
+async fn run_bootstrap_session(
+    mut server: BootstrapServerBinder,
+    controller_tx: mpsc::Sender<BSInternalMessage>,
+    arc_counter: Arc<()>,
+    config: BootstrapConfig,
+    remote_addr: SocketAddr,
+    data_execution: Arc<RwLock<FinalState>>,
+    version: Version,
+    consensus_command_sender: Box<dyn ConsensusController>,
+    network_command_sender: NetworkCommandSender,
+) {
+    debug!("awaiting on bootstrap of peer {}", remote_addr);
+    let res = tokio::time::timeout(
+        config.bootstrap_timeout.into(),
+        manage_bootstrap(
+            &config,
+            &mut server,
+            data_execution,
+            version,
+            consensus_command_sender,
+            network_command_sender,
+        ),
+    )
+    .await;
+    // This drop allows the server to accept new connections without having to complete the error notifications
+    drop(arc_counter);
+    let _ = controller_tx.send(BSInternalMessage::Complete).await;
```

Comment:
consider reverting the introduction of the complete message, and provide the logging in the arc-drop

Reply:
done, though without the the arc-drop logging. That feels like a premature commitment, given the goal on further iterations